utest/shared/src/main/scala/utest/runner/BaseRunner.scala

```diff
@@ -38,7 +38,8 @@ abstract class BaseRunner(val args: Array[String],
                           val remoteArgs: Array[String],
                           testClassLoader: ClassLoader,
                           useSbtLoggers: Boolean,
-                          formatter: utest.framework.Formatter)
+                          formatter: utest.framework.Formatter,
+                          startHeader: Option[String => String])
```

Comment:
Can we can add an overloaded constructor that falls back to the old behavior? I don't know how common it is extend BaseRunner but it seems like we can avoid breaking clients just in case.

Reply:
No problem.

Side note: is there a style guide for utest? It isn't using scalafmt and I wasn't quite sure how to format the secondary constructor.

Reply:
There's isn't any style guide I'm aware of, I think the convention is to align by open paren in definition site but not call site. It's not automatically enforced.